parmed/openmm/topsystem.py

```diff
@@ -230,7 +227,7 @@ def _process_angle(struct, force):
 def _process_urey_bradley(struct, force):
     """ Adds Urey-Bradley parameters to the structure """
     if not struct.angles:
-        warnings.warn('Adding what seems to be Urey-Bradley terms before '
+        warnings.warn('Adding what seems to be Urey-Bradley terms before ' # pragma: no cover
```

Comment:
do you ignore all warnings for coverage? if so, you can add the stuff to .coverc config:

https://github.com/hainm/ParmEd/blob/71e158f731c69a5007e9a507ad304a710cb059d4/test/.coveragerc